### PR TITLE
RSDK-5739 Add Get Analog to board RC card

### PR DESCRIPTION
**Pull request by oliviamiller**
Adds the get analog option to poll the value of an analog pin.

I think we could remove the top analogs section that displays all the analog readers by name and their values now that we have this ? Lmk if I should remove in this PR. 

Screenshot 2023-11-22 at 10 23 34 AM

### CODE REVIEW

**Comment by susmitaSanyal** [comment]
@oliviamiller This is a draft PR. Are you planning on making more changes?

**Comment by oliviamiller** [comment]
> @oliviamiller This is a draft PR. Are you planning on making more changes?

its ready now!

File: components/board/fake/board.go [hunk 1]
```diff
@@ -272,8 +272,10 @@ func (b *Board) SetPowerMode(ctx context.Context, mode pb.PowerMode, duration *t
 	return grpc.UnimplementedError
 }
 
-// WriteAnalog writes the value to the given pin.
+// WriteAnalog writes the value to the given pin. Add to AnalogReaders so you can read the value back.
```
Comment (susmitaSanyal) [inline components/board/fake/board.go:275]:
```suggestion
// WriteAnalog writes the value to the given pin, which can be read back by adding it to AnalogReaders.
```

File: web/frontend/src/components/board/index.svelte [hunk 2]
```diff
@@ -237,31 +251,61 @@ const handleAnalogValueInput = (event:CustomEvent<{ value: string}>) => {
...
+            {readAnalogMessage}
+            </span>
+          </div>
+        </td>
+      </tr>
+      <tr>
+     <th class="border border-medium p-2">
+          Set
+        </th>
+       <td class="border border-medium p-2">
```
Comment (susmitaSanyal) [inline web/frontend/src/components/board/index.svelte:286]: Great job!!! 
Looks like the Analogs box at the top is now redundant. Let's remove it in this PR.

**Review by susmitaSanyal — CHANGES_REQUESTED** [review]

File: web/frontend/src/components/board/index.svelte (hunk 2 above)
Comment (oliviamiller) [reply web/frontend/src/components/board/index.svelte:286]: talked with rand offline, going to keep for now.

**Review by micheal-parks — APPROVED** [review]

**Review by susmitaSanyal — APPROVED** [review]

**Pull request merged**